src/IssueViz.Security/HotspotsList/TableDataSource/HotspotTableEntryWpfElementFactory.cs

```diff
@@ -64,8 +69,7 @@ private ContextMenu CreateContextMenu()
                 Header = "Remove",
                 Icon = new CrispImage { Moniker = KnownMonikers.DeleteTableRow }
             };
-            // TODO: implement the click
-            //removeEntryMenuItem.Click += (sender, args) => removeCallback();
+            removeEntryMenuItem.Click += (sender, args) => removeCallback();
```

Comment:
This is starting to feel a bit ugly and non-WPF-y :-(
I hope we don't have to add any more commands or we might need to have a re-think about how to wire up the commands.